apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -59,9 +62,10 @@ const AssignContentTypeSection = (props: Props) => {
   const sdk = useSDK<AppExtensionSDK>();
 
   useEffect(() => {
+    if (parameters.useTrailingSlash) setUseTrailingSlash(parameters.useTrailingSlash);
     if (parameters.contentTypes) setContentTypes(parameters.contentTypes);
     setLoadingContentTypes(false);
-  }, [parameters.contentTypes]);
+  }, [parameters.contentTypes, parameters.useTrailingSlash]);
```

Comment:
a nit: I feel useTrailingSlash in code looks more like a react hook. Maybe we could name it `forceTrailingSlash` or something like that

Reply:
Good point, totally agree! Updated the name to `forceTrailingSlash`